### PR TITLE
Gateone - Update to git-6ae1d01 and generic DSM5+6 service approach

### PR DESCRIPTION
**Pull request by m4tt075**
## Motivation
The gateone package has been broken since more than two years. Time for an update...
Linked issue: #2340 

## List of updates and fixes
- Updates Gateone package to git-6ae1d01 (20171125)
- Introduces generic DSM5+6 service approach
- Fixes installation of certificates and keyfiles on DSM6 as well as DSM5 systems
- Updates dependency `dtach` cross-package to v0.9

### Checklist
- [x] Build rule `all-supported` completed successfully
- [ ] Package upgrade completed successfully
- [x] New installation of package completed successfully (5.2 and 6.1 firmware)

### CODE REVIEW

**Comment by m4tt075** [comment]
@ymartin59 @Safihre @BenjV Took me three evenings to arrive at this stage. Never dealt with a python package before. Python dependencies were missing (added in `requirements.txt`). The certificate management was broken. Permissions needed fixing as well.

I'm planning to make testbuilds available again, but would appreciate your critical review of the commits before I do that. As always, any advice / improvements will be appreciated...

File: cross/pip/Makefile [hunk 1]
```diff
@@ -1,9 +1,10 @@
 PKG_NAME = pip
-PKG_VERS = 9.0.1
+PKG_VERS = 18.0
```
Comment (ymartin59) [inline cross/pip/Makefile]: I do not understand how pip upgrade is related to GateOne. If a new version is required, I guess Python has to be updated to include newer pip version, right?

File: spk/gateone/Makefile [hunk 2]
```diff
@@ -1,31 +1,37 @@
...
-ADMIN_PORT = 8271
 RELOAD_UI = yes
 DISPLAY_NAME = GateOne
-CHANGELOG = "Update to revision 1d0e803"
+STARTABLE = yes
+CHANGELOG = "1. Update Gateone to revision 6ae1d01, 2. Update dtach to v0.9, 3. Use DSM 5+6 service approach."
+HOMEPAGE = http://liftoffsoftware.com/Products/GateOne
+LICENSE = AGPLv3
+
+# WIZARDS_DIR = src/wizard/
```
Comment (ymartin59) [inline spk/gateone/Makefile]: Have you planned wizard steps?

**Comment by m4tt075** [comment]
OK, squashed and force-pushed the requested changes.
All builds green: https://travis-ci.org/m4tt075/spksrc/builds/421754004
Test-builds here: https://github.com/m4tt075/spksrc/releases/tag/gateone-g6ae1d01

File: cross/pip/Makefile (hunk 1 above)
Comment (Safihre) [reply cross/pip/Makefile]: They changed to a year-based version scheme, but actually the code changes are super minor.

**Comment by m4tt075** [comment]
@ymartin59 About 50 downloads in the meantime. Positive reports in #2340. The current package is broken. I think this is good to go. Please merge and publish at your earliest convenience...

**Comment by m4tt075** [comment]
@ymartin59 Don't know. I haven't changed anything in that regard. A quick search led me here:
https://liftoff.github.io/GateOne/About/configuration.html?highlight=log%20level
Seems "info" is the default log level. We could set it to "warning" if you want...

**Pull request merged**